release-notes-1.4.0.md

```diff
@@ -2,6 +2,7 @@
 ${version-number}
 
 #### New Features
+- [SCMOD-3111](https://jira.autonomy.com/browse/SCMOD-3111): A new feature to allow the storing of tenant specific configurations that can then be used by the workflow worker to create a tenant specific workflow has been added. The functionality for CRUD operations has also been added for this new feature.
```

Comment:
Storing the tenant configs is part of CRUD operations so it sounds odds to say that storing has been implemented and so have CRUD operations. Change to say "to allow the management of tenant specific config" and remove the second sentence.